src/views/Reservations/Reservations.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./Reservations.css";
+
+const Reservations = () => {
+  return (
+    <div className="form-content-right">
+      <form className="form">
+        <h1>Make a reservation</h1>
+        <div className="form-inputs">
+          <label className="form-label">Name</label>
+          <input type="text" name="name" className="form-input" placeholder="Name" />
+        </div>
+        <div className="form-inputs">
+          <label className="form-label">Phone</label>
+          <input type="tel" name="phone" className="form-input" placeholder="Phone" />
+        </div>
+        <div className="form-inputs">
+          <label className="form-label">Date</label>
+          <input type="date" name="date" className="form-input" placeholder="Date" />
+        </div>
+        <div className="form-inputs">
+          <label className="form-label">Email</label>
+          <input type="email" name="email" className="form-input" placeholder="Email" />
+        </div>
+        <div className="form-inputs">
+          <label className="form-label">Time</label>
+          <input type="time" name="time" className="form-input" placeholder="Time" />
+        </div>
+        <div className="form-inputs">
+          <label className="form-label">Number of Persons</label>
+          <input type="number" name="persons" className="form-input" placeholder="Number of Persons" />
```

Comment:
Change the "Number of Persons" `<input>` to "Choose a table" `<select>` dropdown list - where user can choose one of the following options:
"For 8 people"
"For 6 people"
"For 5 people"
"For 4 people"